packages/contracts-bedrock/foundry.toml

```diff
@@ -11,7 +11,7 @@ build_info_path = 'artifacts/build-info'
 snapshots = 'notarealpath' # workaround for foundry#9477
 
 optimizer = true
-optimizer_runs = 999999
+optimizer_runs = 5000
```

Comment:
We will want to start using the feature added in https://github.com/foundry-rs/foundry/pull/8668 to set the `optimizer_runs` on a per contract basis. This will save us from modifying the bytecode of all of the existing contracts with this PR

Reply:
> compilation_restrictions allows users to configure how we compile individual files or directories. Currently accepted keys are min_evm_version, max_evm_version, min_optimizer_runs, max_optimizer_runs, cancun. Configured restrictions apply to sources importing the restricted file as well.